### PR TITLE
Fix various docs

### PR DESCRIPTION
**Pull request by piotmag769**

### CODE REVIEW

File: docs/src/appendix/snforge-library/declare.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 > `fn declare(contract: ByteArray) -> Result<ContractClass, Array<felt252>>`
 
-Declares a contract for later deployment.
+Declares a contract for later deployment. The `contract` is the name of the contract (part after `mod` keyword) e.g. `"HelloStarknet"`.
```
Comment (Arcticae) [inline docs/src/appendix/snforge-library/declare.md:5]: This is not needed, it's documented in cairo

File: docs/src/appendix/snforge-library/declare.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
...
 
-See [docs of `ContractClass`](./contract_class.md) for more info about the resulting struct.
+Check docs of [`ContractClass`](./contract_class.md) for more info about the resulting struct.
```
Comment (Arcticae) [inline docs/src/appendix/snforge-library/declare.md:7]: it's more visible with the fragment hyperlinked

File: docs/src/appendix/snforge-library/fs/read_json.md [hunk 3]
```diff
@@ -25,8 +25,8 @@ fn test_read_json() {
 
 ## Accepted format
 File content must have proper JSON Format with values satisfying the conditions:
-  - integers in range of `[0, P)` where P is [`Cairo Prime`](https://book.cairo-lang.org/ch02-02-data-types.html?highlight=prime#felt-type)
-  - single line strings (`ByteArray`) ie. `"very very very very loooooong string"`, new lines can be used with `\n` and `"` with `\"`
+  - integers, in decimal format, in range of `[0, P)` where P is [`Cairo Prime`](https://book.cairo-lang.org/ch02-02-data-types.html?highlight=prime#felt-type)
```
Comment (Arcticae) [inline docs/src/appendix/snforge-library/fs/read_json.md:28]: the note about decimal format is unneeded, it's a constraint of json files

File: docs/src/appendix/snforge-library/declare.md (hunk 1 above)
Comment (piotmag769) [reply docs/src/appendix/snforge-library/declare.md:5]: Then why do we have this page at all? I think it should be there

File: docs/src/appendix/snforge-library/declare.md (hunk 2 above)
Comment (piotmag769) [reply docs/src/appendix/snforge-library/declare.md:7]: Imo it's better and more consistent this way

File: docs/src/appendix/snforge-library/declare.md (hunk 1 above)
Comment (Arcticae) [reply docs/src/appendix/snforge-library/declare.md:5]: Discoverability, interface

File: docs/src/appendix/snforge-library/declare.md (hunk 2 above)
Comment (Arcticae) [reply docs/src/appendix/snforge-library/declare.md:7]: Not a huge deal for me but please abstain from further correcting this, it's not worth the attention

File: docs/src/appendix/snforge-library/declare.md (hunk 1 above)
Comment (piotmag769) [reply docs/src/appendix/snforge-library/declare.md:5]: Imo it should be here, but up 2 u

**Review by Arcticae — APPROVED** [review]

**Pull request merged**